### PR TITLE
Fix preference to accept arrays

### PR DESCRIPTION
**Pull request by jordanwade**
### 1. What does this PR do, or why is it needed?
Fixes preferences to be able to have an array of preferences instead of a single string. This also fixes wide open search by fixing this bug.

### 2. How do I test this PR?

Check group search and make sure everything is working as expected. Also, wide open search should return all groups again.

## SCREENSHOTS/GIFS

| Web |
| --- |
| image |

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [tag-issues-here]
- [x] No new warnings in tests, storybook, or apps
- [x] Assign relevant reviewers
- [x] Upload GIF(s) of iOS, Android, Web
- [ ] Get one peer approval before merging

## REVIEW:

**Manual QA**

- [ ] QA branch on iOS and ensure it looks/behaves as expected
- [ ] QA branch on Android and ensure it looks/behaves as expected
- [ ] QA branch on Web and ensure it looks/behaves as expected

**Code Review**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: src/data/group-item/schema.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const groupSchema = gql`
     schedule: Schedule
     videoCall: VideoCallParams
     campus: Campus
-    preference: String
+    preference: [String]
```
Comment (rkdavidson) [inline src/data/group-item/schema.js:93]: Hesitant to suggest this, but... should we semantically go with `preferences` now, too? 🤔

Comment (jordanwade) [reply src/data/group-item/schema.js:93]: I agree, I'll do a follow up PR for that. We also will need to update subPreferences like this, but again I was going to do a PR for that too. 👍

**Review by rkdavidson — APPROVED** [review]
